components/BottomAppBar/README.md

```diff
@@ -115,3 +119,62 @@ tracking its development:
 - Task: [Implement a color themer](https://www.pivotaltracker.com/story/show/157095394)
 - Task: [Impement a surface variant themer](https://www.pivotaltracker.com/story/show/156933141)
 
+
+## Accessibility
+
+<!-- Extracted from docs/accessibility.md -->
+
+To help ensure your bottom app bar is accessible to as many users as possible, please be sure to review the
+following recommendations:
+
+### Set `-accessibilityLabel`
+
+Set an appropriate
+[`accessibilityLabel`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619577-accessibilitylabel)
+to all buttons within the bottom app bar.
+
+#### Swift
+
+```swift
+bottomAppBar.floatingButton.accessibilityLabel = "Compose"
+```
+
+#### Objective-C
+
+```objc
+bottomAppBar.floatingButton.accessibilityLabel = "Compose"
+```
+
+If you do not do this then the button's label will be the name of the image.
+
+#### Swift
+
+```swift
+let image = UIImage(named: "Edit")
+bottomAppBar.floatingButton.setImage(image, for: .normal)
+```
+
+#### Objective-C
+```objc
+UIImage *image = [UIImage imageNamed:@"Edit"];
+[bottomAppBar.floatingButton setImage:image forState:UIControlStateNormal];
+```
```

Comment:
I can't tell what these two blocks of code are demonstrating accessibility-wise - can we remove them?

Reply:
I was trying to say if you don't set an accessibilityLabel then it would read out "Edit button"